chapters/BP_064_en.md

```diff
@@ -0,0 +1,40 @@
+## Cache often-used computed Data
+
+### Identifiers
+
+| GreenIT | V2  | V3  |  V4  |
+|:-------:|:---:|:---:|:----:|
+|   47    | 62  | 64  |      |
+
+### Categories
+
+| Life cycle |  Tiers  |         Responsible          |
+|:----------:|:-------:|:----------------------------:|
+| 2. Design  | Network | Software Architect/Developer |
+
+### Indications
+
+|      Priority      | Implementation difficulty | Ecological impact |
+|:------------------:|:-------------------------:|:-----------------:|
+|         4          |             3             |         4         |
+
+|                      Saved resources                      |
+|:---------------------------------------------------------:|
+|                      Processor / RAM                      |
+
+### Description
+
+When values or data calculations are resources intensive, you should cache the results when the values remain unchanged, to voir repeating these operations. 
```

Comment:
```suggestion
When values or data calculations are resources intensive, you should cache the results when the values remain unchanged, to avoid repeating these operations. 
```